tools/api_versioning/utils.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+"""Utility functions for accessing the API version (X.Y.Z format).
+"""
+from collections import namedtuple
+import pathlib
+
+ApiVersion = namedtuple('ApiVersion', ['major', 'minor', 'patch'])
+
+
+# Errors that happen during API version parsing.
+class ApiVersionParsingError(Exception):
+    pass
+
+
+def get_api_version(input_path):
+    """Returns the API version from a given API version input file.
+
+    Args:
+      input_path: the file containing the API version (API_VERSION).
+
+    Returns:
+      a namedtuple containing the major, minor, patch versions.
```

Comment:
Nit: `Throws` docsstring

Reply:
Added the `Raises` clause to the docstring.